CONTRIBUTING.md

```diff
@@ -44,30 +44,43 @@ are more sensitive, emailed to <opensource@google.com>.
 
   1. Go makes it very simple to ensure properly formatted code, so always run
      `go fmt` on your code before committing it. You should also run
-     [golint][] over your code. As noted in the [golint readme][], it's not
-     strictly necessary that your code be completely "lint-free", but this will
-     help you find common style issues.
+     [go vet][] over your code. this will help you find common style issues
+     within your code and will keep styling consistent within the project.
 
-  1. Any significant changes should almost always be accompanied by tests. The
+  2. Any significant changes should almost always be accompanied by tests. The
      project already has good test coverage, so look at some of the existing
      tests if you're unsure how to go about it. [gocov][] and [gocov-html][]
      are invaluable tools for seeing which parts of your code aren't being
      exercised by your tests.
 
-  1. Please run:
+  3. Please run:
      * `go generate github.com/google/go-github/...`
      * `go test github.com/google/go-github/...`
      * `go vet github.com/google/go-github/...`
 
-  1. Do your best to have [well-formed commit messages][] for each change.
+   The `go generate ./...` command will update or generate certain files, and the resulting changes should be included in your pull request.
+
+   The `go test ./...` command will run tests inside your code. This will help you spot places where code might be faulty before commiting.
```

Comment:
```suggestion
   The `go test ./...` command will run tests inside your code. This will help you spot places where code might be faulty before committing.
```